### DISCUSSION TITLE
how to view 3 or more pages at once side by side?

### DISCUSSION THREAD

**Discussion by imcool** [sumatrapdfreader/sumatrapdf#4013] [Q&A]
how to view 3 or more pages at once side by side?

I tried adding this in settings file but doest not work, no change, i want to view page1,2,3,4 at once from a pdf.

FixedPageUI [
	TextColor = #000000
	BackgroundColor = #ffffff
	SelectionColor = #f5fc0c
	WindowMargin = 2 4 2 4
	PageSpacing = 4 4
	InvertColors = false
	HideScrollbars = false
	ColumnsPerPage = 2
]

**Comment by GitHubRulesOK** [comment]
You can set screen to 1 or 2 pages wide but there is no 3 or 4 see https://github.com/sumatrapdfreader/sumatrapdf/issues/246 and https://github.com/sumatrapdfreader/sumatrapdf/issues/1171
